<h1> Title </h1>

<h4> By: James Magee, Benjamin Nguyen, Shubham Sharma, Nav Thukral </h4>

<br>

<h2> Introduction </h2>

Begin by providing some relevant background information on the topic so that someone unfamiliar with it will be prepared to understand the rest of your proposal.

Clearly state the question you will try to answer with your project. Your question should involve one or more random variables of interest, spread across two or more categories that are interesting to compare. For example, you could consider the annual maxima river flow at two different locations along a river, or perhaps gender diversity at different universities. Of the response variable, identify one location parameter (mean, median, quantile, etc.) and one scale parameter (standard deviation, inter-quartile range, etc.) that would be useful in answering your question. Justify your choices.

<br>

<h2> Preliminary Results </h2>

In this section, you will:

- Demonstrate that the dataset can be read from the web into R.
- Clean and wrangle your data into a tidy format.
- Plot the relevant raw data, tailoring your plot in a way that addresses your question.
- Compute estimates of the parameter you identified across your groups. Present this in a table. If relevant, include these estimates in your plot.

Be sure to not print output that takes up a lot of screen space.

<h3> Loading necessary libraries </h3>

In [1]:
#import necessary libraries, suppressing messages
suppressMessages(library(tidyverse))
suppressMessages(library(readr))
suppressMessages(library(tidyr))
suppressMessages(library(dbplyr))
suppressMessages(library(ggplot2))

#set seed to ensure reproducibility of results
set.seed(9999)

<h3> Reading training data into R</h3>

In [2]:
training_data <- read_csv("train.csv")
head(training_data)

Rows: 891 Columns: 12
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (5): Name, Sex, Ticket, Cabin, Embarked
dbl (7): PassengerId, Survived, Pclass, Age, SibSp, Parch, Fare

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


PassengerId Survived Pclass
1 1           0        3     
2 2           1        1     
3 3           1        3     
4 4           1        1     
5 5           0        3     
6 6           0        3     
  Name                                                Sex    Age SibSp Parch
1 Braund, Mr. Owen Harris                             male   22  1     0    
2 Cumings, Mrs. John Bradley (Florence Briggs Thayer) female 38  1     0    
3 Heikkinen, Miss. Laina                              female 26  0     0    
4 Futrelle, Mrs. Jacques Heath (Lily May Peel)        female 35  1     0    
5 Allen, Mr. William Henry                            male   35  0     0    
6 Moran, Mr. James                                    male   NA  0     0    
  Ticket           Fare    Cabin Embarked
1 A/5 21171         7.2500 NA    S       
2 PC 17599         71.2833 C85   C       
3 STON/O2. 3101282  7.9250 NA    S       
4 113803           53.1000 C123  S       
5 373450            8.0500 NA    S       
6 330877            8.4583 NA    Q

We can see that the training data contains 891 observations and 12 variables, with many NA values. Even though the data is already in tidy format, to further serve the purposes of our analysis, we will need to split the dataframe into two, according to whether each passenger survived or not (via the `Survived` column). We can then use the two dataframes to compare statistics between them.

<h3> Sorting data according to survival </h3>

In [4]:
#Filter training data to only include columns with people that did not survive (Survived == 0)
training_data_survived_no <- training_data %>%
    filter(Survived == 0)

#Filter training data to only include columns with people that survived (Survived == 1)
training_data_survived_yes <- training_data %>%
    filter(Survived == 1)

In [5]:
#Display first 6 observations of training_data_survived_no
head(training_data_survived_no)

PassengerId Survived Pclass Name                           Sex  Age SibSp
1  1          0        3      Braund, Mr. Owen Harris        male 22  1    
2  5          0        3      Allen, Mr. William Henry       male 35  0    
3  6          0        3      Moran, Mr. James               male NA  0    
4  7          0        1      McCarthy, Mr. Timothy J        male 54  0    
5  8          0        3      Palsson, Master. Gosta Leonard male  2  3    
6 13          0        3      Saundercock, Mr. William Henry male 20  0    
  Parch Ticket    Fare    Cabin Embarked
1 0     A/5 21171  7.2500 NA    S       
2 0     373450     8.0500 NA    S       
3 0     330877     8.4583 NA    Q       
4 0     17463     51.8625 E46   S       
5 1     349909    21.0750 NA    S       
6 0     A/5. 2151  8.0500 NA    S

Table 1: Training data of people that did not survive

In [6]:
#Display first 6 observations of training_data_survived_yes
head(training_data_survived_yes)

PassengerId Survived Pclass
1  2          1        1     
2  3          1        3     
3  4          1        1     
4  9          1        3     
5 10          1        2     
6 11          1        3     
  Name                                                Sex    Age SibSp Parch
1 Cumings, Mrs. John Bradley (Florence Briggs Thayer) female 38  1     0    
2 Heikkinen, Miss. Laina                              female 26  0     0    
3 Futrelle, Mrs. Jacques Heath (Lily May Peel)        female 35  1     0    
4 Johnson, Mrs. Oscar W (Elisabeth Vilhelmina Berg)   female 27  0     2    
5 Nasser, Mrs. Nicholas (Adele Achem)                 female 14  1     0    
6 Sandstrom, Miss. Marguerite Rut                     female  4  1     1    
  Ticket           Fare    Cabin Embarked
1 PC 17599         71.2833 C85   C       
2 STON/O2. 3101282  7.9250 NA    S       
3 113803           53.1000 C123  S       
4 347742           11.1333 NA    S       
5 237736           30.0708 NA    C       
6 PP 9549          16.7000 G6    S

Table 2: Training data of people that did survive

One interesting fact to notice right away is that all of the first 6 observations of people that *did not survive* are **male**, while all of the first 6 observations of people that *did survive* are **female**. This makes it seem as though there is a correlation between gender and survivability during the sinking of the Titanic. This will be examined further.

<br>

<h2> Methods: Plan </h2>

The previous sections will carry over to your final report (you’ll be allowed to improve them based on feedback you get). Begin this Methods section with a brief description of “the good things” about this report – specifically, in what ways is this report trustworthy?

Continue by explaining why the plot(s) and estimates that you produced are not enough to give to a stakeholder, and what you should provide in addition to address this gap. Make sure your plans include at least one hypothesis test and one confidence interval. If possible, compare both the bootstrapping and asymptotics methods.

Finish this section by reflecting on how your final report might play out:

- What do you expect to find?
- What impact could such findings have?
- What future questions could this lead to?

<br>

<h2> References </h2>

At least two citations of literature relevant to the project. The citation format is your choice – just be consistent. Make sure to cite the source of your data as well.